src/drafty.js

```diff
@@ -1224,9 +1399,61 @@ Drafty.UNSAFE_toHTML = function(content) {
   return txt;
 }
 
+// Denormalizes formatting and entity data into a simple span array.
+function prepareSpans(fmt, ent) {
```

Comment:
There is `spannify` and `prepareSpans` while spans they produce are completely different. Maybe generate the same format of spans or somehow un-confuse the method names?